.nvmrc

```diff
@@ -0,0 +1 @@
+19.8.1
```

Comment:
is this saying use node 19?

Reply:
seem pretty specific

Reply:
If you use `nvm` then it will default to this version